### PR TITLE
feat(core-select): Add custom search delimiter for apiV4

### PR DESCRIPTION
**Pull request by JeanBeaurepaire**
## Description

Allow to use searchDelimiter input for apiV4 select to search with comma delimiter for example.
By default it's space delimiter

-----

Optionally, technical or more in-depth description for reviewers.
Keep an empty line under your text, as well as the 5 lines that follow it.

-----

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-3115/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-3115/storybook

**Review by Supamiu — DISMISSED** [review]

File: stories/documentation/forms/select/custom-api-example.component.ts [hunk 1]
```diff
@@ -15,6 +15,6 @@ export interface Legume {
 export class LuCoreSelectLegumesDirective extends LuCoreSelectApiV4Directive<Legume> {
 	public constructor() {
 		super();
-		this.apiV4 = '/api/legumes';
+		this.apiV4 = input('/api/legumes');
```
[the remark this answers is not in the document]
Comment (JeanBeaurepaire) [reply stories/documentation/forms/select/custom-api-example.component.ts]: En effet, merci

**Review by GuillaumeNury — APPROVED** [review]

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-3115/storybook

**Review by GuillaumeNury — APPROVED** [review]

**Review by GuillaumeNury — APPROVED** [review]

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-3115/storybook

**Review by jeremie-lucca — APPROVED** [review]

**Pull request merged**